sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
+These shared APIs allow client libraries to expose common functionality in a consistent fashion, 
+so that once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
 
-# Installing
+The main shared concepts of Azure.Core (and so all Azure SDK libraries using Azure.Core) include:
+
+1. Unified APIs for configuring service clients, e.g. configuring retries, logging.
+2. Unified APIs for accessing HTTP responses.
+3. Unified APIs for consuming long running operations (LROs).
+4. Unified APIs for consuming asynchronous streams (```IAsyncEnumerable<T>```) by paging.
```

Comment:
nit:  Single ticks would do just fine here, as this is an inline term and not an expanded snippet.

Reply:
I removed "Unified"